cms/templates/bulksettings.html

```diff
@@ -59,23 +59,36 @@ <h4 class = "section-header-title" title="Collapse/Expand this utility">
                                 <td class = "subsection-header-cell" colspan = "7">
                           <strong><a href="${subsection_settings['url']}">${subsection_settings['name']}</a></strong> -
                           % for setting_type in setting_type_list_map['subsection_setting_types']:
-                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_settings[setting_type]}</span>
+                              <%
+                              subsection_setting = subsection_settings[setting_type] or "N/A"
+                              %>
+                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_setting}</span>
                           % endfor
-                        </td>
+                                </td>
```

Comment:
Sorry - the </td> didn't look aligned with the <td> for subsection-header-cell.  Should I move the </td> back to its old spot?